content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -262,6 +262,11 @@ Secret somewhere that your terminal / computer screen could be seen by an onlook
 When you delete a ServiceAccount that has an associated Secret, the Kubernetes
 control plane automatically cleans up the long-lived token from that Secret.
 
+{{< note >}}
+Manually created Secret API objects do not appear in the service account API object's
+`.secrets` field. That field is only populated with auto-generated secrets.
+{{< /note >}}
```

Comment:
The phenomenon you described is true, but the way you describe it is confusing.

Reply:
Thanks @tengqm for taking look at this.

Reply:
can we write like this:

> If you view the ServiceAccount using:
> 
> ` kubectl get serviceaccount build-robot -o yaml`
> 
> You can't see the `build-robot-secret` secret in the service account API objects [.secrets](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/service-account-v1/) field because Manually created Secret API objects do not appear in the service account API object's `.secrets` field. That field is only populated with auto-generated secrets.

In this, we use the same information for Secrets and SA which are present on the corresponding page.

Your thoughts on this?

Reply:
This revision is better. Please pay attention to how you spell 'Secret' as a resource type. Please also try using "ServiceAccount" rather than "service account" when appropriate.

Reply:
Sure, I will update it accordingly.

Reply:
Addressed the comment, PTAL!